### ISSUE TITLE
Pakket::Scaffolder::Perl::get_release_info() sometimes returns wrong result         

### ISSUE THREAD

**Issue by zprogd** [xsawyerx/pakket#155]
When we call get_release_info() and using distribution name($name='Search-Elasticsearch') and $requirements have modules names ("Search::Elasticsearch"), function returns wrong result, last version for that distribution name, without checking requirements.
because  in $requirements->accepts_module()
"Search::Elasticsearch" != 'Search-Elasticsearch'

**Comment by zprogd** [comment]
How to reproduce: add any package with prerequisite which has two_words_name(Search-Elasticsearch) and not the last version requirement. Scaffolder will use the last version for that prerequisite.
Example:
`pakket manage add perl/MetaCPAN-Client=1.028002`

pakket_lib/SPECS/index.json:
```
"perl/MetaCPAN-Client=1.028002:1" : "dea8a1b5f8bf25ea48e8072b1727e6cf61a3e5ea.sgm",
"perl/Search-Elasticsearch=5.02:1" : "9fd3ccf9c0b0eed72f7db4f0afd99f8095b26456.sgm",
```

pakket_lib/SPECS/dea8a1b5f8bf25ea48e8072b1727e6cf61a3e5ea.sgm:
```
"Search-Elasticsearch" : {
                "version" : ">= 2.02, < 5.00"
},
```
That means that now we are using only last versions for all two_words prerequisites.

**Comment by mickeyn** [comment]
fixed by 0d18b8f7fe6e7be7dde9b3e269f2a495e0b5f415
